src/app/icd/server/ICDManager.cpp

```diff
@@ -37,6 +37,8 @@ using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::IcdManagement;
 using namespace System::Clock;
 
+#define TOGGLE_STATE(state) ((state == OperationalState::IdleMode) ? OperationalState::ActiveMode : OperationalState::IdleMode)
```

Comment:
can we do this typesafe?

Even an `void toggleState(OperationalState &state)` seems nicer.

Reply:
is this used?

Reply:
Was initial used but not in the last commit. Forgot to remove it.

Reply:
removed.